### ISSUE TITLE
Trailing slash is left in proxy port when parsing http_proxy / https_proxy env var

### ISSUE THREAD

**Issue by gergelyfabian** [bazelbuild/rules_jvm_external#321]
https://github.com/bazelbuild/rules_jvm_external/blob/b0c23d8260a0b3aff27982a0bc2f397919a84aea/private/proxy.bzl#L27

When your environment is configured to use a proxy in a similar way (that should be the standard way to do):

export http_proxy=http://127.0.0.1:3128/

Running `bazel run @unpinned_maven//:pin` fails with timeout (as we have been guessing, because it cannot find the proxy).
Debugging rules_jvm_external showed that proxy.bzl / _get_proxy_port() has a bug, because it is not cutting out trailing slash from the url and is leaving it in the returned port number (e.g. "3128/").
Then coursier's java options are set up with "3128/" port number that won't work.

Adding `print("Command: %s" % cmd)` to the end of coursier.bzl / _generate_java_jar_command() prints:

rules_jvm_external/coursier.bzl:90:5: Command: [/usr/lib/jvm/java-1.8.0-openjdk-amd64/bin/java, "-noverify", "-jar", "-Dhttp.proxyHost=127.0.0.1", "-Dhttp.proxyPort=3128/", "-Dhttps.proxyHost=127.0.0.1", "-Dhttps.proxyPort=3128/", "-Dhttp.nonProxyHosts=localhost|127.0.0.0/8|::1", /home/user/.cache/bazel/_bazel_user/c82d5fe1f9e3c22172c725eae3cdb8bf/external/unpinned_maven/coursier]

Switching line 27 of proxy.bzl to the following fixes this issue:

```
netloc = url.split("://", 1)[1].split("/")[0].split("@")[-1]
```

`.split("/")[0]` is the new code.

**Comment by jin** [comment]
Fixed by #322

**Comment by gergelyfabian** [comment]
Thanks @jin!
